jaeger/cmd/install.go

```diff
@@ -97,19 +97,52 @@ func render(w io.Writer, values *jaeger.Values) error {
 		)
 	}
 
-	chart := &charts.Chart{
-		Name:      "jaeger",
-		Dir:       "jaeger",
-		Namespace: values.Namespace,
-		RawValues: rawValues,
-		Files:     files,
-		Fs:        static.Templates,
+	var partialFiles []*loader.BufferedFile
+	for _, template := range charts.L5dPartials {
+		partialFiles = append(partialFiles,
+			&loader.BufferedFile{Name: template},
+		)
+	}
+
+	// Load all jaeger chart files into buffer
+	if err := charts.FilesReader(static.Templates, "jaeger/", files); err != nil {
+		return err
+	}
+
+	// Load all partial chart files into buffer
+	if err := charts.FilesReader(partials.Templates, "", partialFiles); err != nil {
+		return err
+	}
+
+	// Create a Chart obj from the files
+	chart, err := loader.LoadFiles(append(files, partialFiles...))
+	if err != nil {
+		return err
 	}
-	buf, err := chart.Render()
+
+	vals, err := chartutil.CoalesceValues(chart, rawValues)
 	if err != nil {
 		return err
 	}
 
+	// Attach the final values into the `Values` field for rendering to work
+	finalValues := make(map[string]interface{})
+	finalValues["Values"] = vals
+
+	renderedTemplates, err := engine.Render(chart, finalValues)
```

Comment:
instead of creating a named variable for the final values, I'd consider just passing a literal:

```
Render(chart, map[string]interface{} { "Values": vals })
```